package.json

```diff
@@ -11,7 +11,7 @@
     "node-test": "mocha node-tests/**/*-test.js"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": ">= 10.*"
```

Comment:
Any particular reason why the minimum Node version must be increased?

Reply:
This version of Babel doesn’t work with 6, and 10 is the latest version of nice still supported

Reply:
OK!